cnf-tests/GATEKEEPER.md

```diff
@@ -0,0 +1,136 @@
+# Gatekeeper testing
+
+This document describes how to test set up and test Gatekeeper manually.
```

Comment:
can you specify here that the setup is via the operator and maybe link to their README?

Reply:
done